### DISCUSSION TITLE
`CANNOT_LINK_EXECUTABLE` using latest version.

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [termux/termux-app#3617] [Q&A] [answered]
I'm getting https://github.com/termux/termux-app/issues/1703#issue-676231188 on newly installed https://f-droid.org/repo/com.termux_118.apk. Isn't 118 the latest version?

**Comment by roke-julian-lockhart** [comment]
> https://github.com/termux/termux-app/discussions/3617#discussion-5693892

@sylirre, why did you close this without explanation?

**Reply by sylirre** [reply, marked as answer]
Because the issue does not really exist.

Install v0.118 from F-Droid and do `pkg upgrade` first before using it.

If have repo availability issues do `termux-change-repo`.

That's all what needed to have Termux working properly.
